corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +46,16 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_web_user_usercases_if_applicable(user, spawn_task):
```

Comment:
Should this function also check `settings.UNIT_TESTING` like `sync_usercases_if_applicable` does?

This and `sync_usercases_if_applicable` are right on the borderline where it might be better to collapse them into one function. `user.get_domains()` should also work for mobile users, it'll just return their one domain.

Reply:
Looking at the test failures, checking `UNIT_TESTING` here will likely fix the one that says `Database queries to 'default' are not allowed in SimpleTestCase subclasses.` - that happens when you try to access the db in a test that doesn't allow it, which sounds like some test that didn't previously use the db is now trying to save usercases.